packages/react/src/components/DataTable/__tests__/__snapshots__/TableToolbarSearch-test.js.snap

```diff
@@ -61,7 +61,7 @@ exports[`TableToolbarSearch renders as expected - Component API should render 1`
         xmlns="http://www.w3.org/2000/svg"
       >
         <path
-          d="M17.4141 16L24 9.4141 22.5859 8 16 14.5859 9.4143 8 8 9.4141 14.5859 16 8 22.5859 9.4143 24 16 17.4141 22.5859 24 24 22.5859 17.4141 16z"
```

Comment:
Any thoughts on why this snapshot is changing? I'd think this would've remained unchanged. It's the svg inside the close button in the TableToolbarSearch